prepare/cards/20_newsgroups.py

```diff
@@ -53,7 +55,20 @@
         ),
     ],
     task="tasks.classification.multi_class",
-    templates="templates.classification.multi_class.all",
+    templates=[
+        StandardTemplate(
+            input_format="Text: {text}",
+            output_format="{label}",
+            target_prefix="Topic: ",
+            instruction=f"Classify the topic of following text to one of these options: {list(map_labels.values())}.",
```

Comment:
I think it should also consume the fields of the task and not to be hard coded: 
```suggestion
            instruction=f"Classify the topic of following text to one of these options: {classes}.",
```
or in whatever way it was in the old templates

Reply:
A neat idea!  I made instruction a formatting string, in analogy with input_format and output_format. 
tweaked the card some, by adding (temporarily) a few loc-s that print out the prompts (